package.json

```diff
@@ -33,9 +34,10 @@
     "@jokeyrhyme/pify-fs": "1.0.1",
     "archiver": "1.1.0",
     "aws-sdk": "^2.5.3",
+    "chalk": "^1.1.3",
     "elegant-spinner": "^1.0.1",
     "execa": "^0.4.0",
-    "inquirer": "^1.0.3",
+    "inquirer": "^1.2.1",
```

Comment:
That sounds like a good plan to me, CLI === looked dependencies, libraries === ranged dependencies